addons/paint-snap/updateScaleTool.js

```diff
@@ -200,7 +200,7 @@ export const updateScaleTool = (paper, tool) => {
     axisLineY.visible = false;
   };
 
-  scaleTool.constructor.prototype.onMouseDrag = function (event) {
+  addons.paintSnap = function (event) {
```

Comment:
What does currently happen if this function is called while `paint-snap` is dynamically disabled? (doesn't look like this PR is changing the behavior, though)

Reply:
I don't really have an idea, but it doesn't seem to break anything.